benchmarking/chain-sync/benchmark-chain-sync-mainnet.sh

```diff
@@ -20,7 +20,7 @@ ulimit -t $CPU_TIME_LIMIT
 
 date --iso-8601=seconds > STARTTIME
 
-NODE="cabal new-run exe:cardano-node -- "
+NODE="cabal new-run exe:cardano-node -- mock-protocol "
```

Comment:
I don't understand. This is for the mainnet.

Reply:
Same as: https://github.com/input-output-hk/cardano-node/pull/335#discussion_r351763574